Simplenote/Classes/SPAuthViewController.swift

```diff
@@ -599,6 +679,22 @@ private enum AuthenticationStrings {
 }
 
 
+// MARK: - PasswordInsecure Alert Strings
+//
+private enum PasswordInsecureString {
+    static let cancel = NSLocalizedString("Cancel", comment: "Cancel Action")
+    static let reset = NSLocalizedString("Reset", comment: "Reset Action")
+    static let title = NSLocalizedString("Error", comment: "Error Title")
+    static let message = [
+        NSLocalizedString("Your password is insecure and must be reset. The password requirements are:", comment: "Password Requirements: Title"),
+        String.newline,
+        NSLocalizedString("- Password cannot match email", comment: "Password Requirement: Email Match"),
+        NSLocalizedString("- Minimum of 8 characters", comment: "Password Requirement: Length"),
+        NSLocalizedString("- No tabs nor newlines allowed", comment: "Password Requirement: Special Characters")
```

Comment:
Muuuuuuch better, thank you!! 

Will loop back with the team, since Android has the same alert. For now going with your first suggestion (Neither). **Thank you thank you!!!**